src/js/date.js

```diff
@@ -47,6 +51,7 @@ function  getMom(ds) {
 }
 
 function normalizeDateString(ds) {
+    if ( ds === null ) { return "N/A"; }
```

Comment:
it's probably more robust to use "if (!ds)" which will cover other untruthy values like undefined.